master/introduction/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: About Calico
+---
+
+Calico provides secure networking for containers and
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other. 
+In private clouds, it can run without IP encapsulation or 
```

Comment:
Strictly speaking Calico can do this in some public clouds as well, so this statement feels a bit misleading.

Reply:
This should be fixed in my update.